discojs/discojs-core/src/aggregator/mean_agg.ts

```diff
@@ -0,0 +1,36 @@
+import { Map } from 'immutable'
+import { Base as Client } from '../client/base'
+
+import { Base as Aggregator } from './base'
+import { Task, TrainingInformant, WeightsContainer, WeightsMapping, aggregation, tf } from '..'
+
+/**
+ * Aggregator that computes the mean of the weights received from the peers.
```

Comment:
is this one used both for secure and cleartext? somehow i don't see the refactor of the SecAgg parts well yet (and trying to keep aggregators compatible with both federated and decentralized as much as possible)

also the robust one by francesco, will it also build on this same class here, or how do they fit together?

Reply:
As mentionned above. Secure aggregation will be brought by @swaglid. Maybe he can push to this PR to not break any already existing feature, or it can be brought by a separate PR.

I have not caught up with Francesco's work. With @s314cy, we've agreed to first focus on getting this PR to pass tests locally, and then to resolve the pretty big conflict with Francesco's PR.